src/commands/ImportUriHandler.ts

```diff
@@ -0,0 +1,41 @@
+import * as querystring from 'querystring';
+import { Disposable, Uri, UriHandler, window } from 'vscode';
+import { ext } from 'vscode-azureappservice/out/src/extensionVariables';
+import { callWithTelemetryAndErrorHandling, IActionContext } from '../../extension.bundle';
+import { localize } from '../localize';
+import { importTrialApp } from './trialApp/importTrialApp';
+
+export class ImportUriHandler implements UriHandler {
```

Comment:
nit: Let's name this to be `ImportTrialAppUriHandler`.  I think it should be more specific since it's being used for only that purpose.